vnext/overrides.json

```diff
@@ -10,7 +10,8 @@
     "DeforkingPatches/README.md",
     "src/tsconfig.json",
     "**/*.windesktop.js",
-    "src/RNTester/js/examples-win/**"
+    "src/RNTester/js/examples-win/**",
+    "src/IntegrationTests/*Test.js"
```

Comment:
Please remove this. We want to track any forking that happens to tests. If there is anything new, we can add it as a platform override.

I expect the new tests to be moved elsewhere though when things are using the new IT framework.

Reply:
Noted. I added this in an earlier PR, because validations were failing if I omitted those new files.

I just noticed the exclude pattern, which should make these entries unnecessary.

Reply:
Done. I'm scoping down to specific test files added by us.